libs/langchain/langchain/document_loaders/parsers/language/cobol.py

```diff
@@ -0,0 +1,58 @@
+import re
+from typing import List
+
+from langchain.document_loaders.parsers.language.code_segmenter import CodeSegmenter
+
+
+class CobolSegmenter(CodeSegmenter):
```

Comment:
Could you add a few unit tests for the segmenter?

Reply:
done